sycl/plugins/cuda/pi_cuda.cpp

```diff
@@ -8,14 +8,15 @@
 
 #include <CL/sycl/backend/cuda.hpp>
 #include <CL/sycl/detail/pi.hpp>
-#include <pi_cuda.hpp>
 #include <algorithm>
 #include <cassert>
 #include <cuda.h>
 #include <cuda_device_runtime_api.h>
-#include <memory>
 #include <limits>
+#include <memory>
 #include <mutex>
+#include <pi_cuda.hpp>
```

Comment:
This looks like a [Main Module Header](https://llvm.org/docs/CodingStandards.html#main-module-header) and therefore, must be included first

Reply:
Though you are completely right, moving it up makes the clang-format-check fail.

Reply:
Please, separate them by empty line

Reply:
Worked like a charm, thank you.